pkg/installmanager/installmanager.go

```diff
@@ -849,14 +866,36 @@ func (m *InstallManager) gatherLogs(cd *hivev1.ClusterDeployment, sshPrivKeyPath
 			m.log.WithError(err).Warn("error fetching logs from bootstrap node")
 			return
 		}
+
 		m.log.Info("successfully gathered logs from bootstrap node")
 	} else {
 		if err := m.gatherClusterLogs(cd); err != nil {
 			m.log.WithError(err).Warn("error fetching logs with oc adm must-gather")
 			return
 		}
+
 		m.log.Info("successfully ran oc adm must-gather")
 	}
+
+	// At this point, all log files are in m.LogsDir
+	// Gather the filenames
+	files, err := ioutil.ReadDir(m.LogsDir)
+	if err != nil {
+		m.log.WithError(err).WithField("clusterprovision", types.NamespacedName{Name: provision.Name, Namespace: provision.Namespace}).Error("error reading Logsdir")
+		return
+	}
+
+	filepaths := []string{}
+	for _, file := range files {
+		if !file.IsDir() {
+			filepaths = append(filepaths, filepath.Join(m.LogsDir, file.Name()))
+		}
+	}
```

Comment:
this seems like it will only include the first level of files in the LogsDir, is that what is expected? shouldn't we upload all the files from all the children dirs too? or atleast log that we are skipping those dirs?

Reply:
When I tested this, the only directory was `lost+found`. That's why this check is in there. I'll add a log message when skipping.